spec/controllers/verify/review_controller_spec.rb

```diff
@@ -38,8 +38,7 @@
         :before,
         :confirm_two_factor_authenticated,
         :confirm_idv_session_started,
-        :confirm_idv_steps_complete,
-        :confirm_idv_attempts_allowed
```

Comment:
the new, consolidated feature test that @monfresh suggested revealed another edge case in the IdV attempt counting, where the 3rd attempt at step 1 was successful, but the check in step 2 considered the attempts exceeded the max because the comparison is `attempts >= max`.

i.e.
attempt step 1 - fail == 1
attempt step 1 - fail == 2
attempt step 1 - ok == 3
load step 2 with check, attempts now == max (3)